org.mixedrealitytoolkit.core/Interactors/IGazeInteractor.cs

```diff
@@ -15,4 +15,4 @@ public interface IGazeInteractor : IXRInteractor
     {
 
     }
-}
\ No newline at end of file
+}
```

Comment:
nit:  minor thing, but would it be possible to remove any of these extraneous line breaks?  There are a few throughout.

Reply:
Or are these being done automatically for any files that don't already have one at the end?

Reply:
I agree they are annoying but those are added by Visual Studio whenever it saves a file.  I even tried [this](https://stackoverflow.com/questions/64501728/how-to-stop-visual-studio-from-putting-extra-newline-at-the-end-of-a-file-on-sav) to make it stop but Visual Studio still adds them whenever it saves a file.  Sorry about that.